src/services/activityReportApprovers.js

```diff
@@ -16,11 +16,22 @@ export async function upsertApprover(values) {
     paranoid: false,
   });
 
-  if (approver && status) {
-    await approver.update({
-      status,
-      note: values.note,
-    }, {
+  if (approver) {
```

Comment:
this is the actual bug. Newly submitted reports don't have an approver status. So if you
1) added an approver
2) saved
3) submitted report

You would trigger this, as the approver hooks would not refire. With this bugfix, everytime we call syncApprovers, they will ALWAYS fire

Reply:
the bug being the report would then not be moved to a calculatedStatus of "submitted"